From e38826920ff1f2927b63f6c51663197fa147fd0e Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Mon, 23 Feb 2015 22:18:27 +0000
Subject: [PATCH 4/6] lang: Add to make (^Makefile, ^Makefile.Debug,
 ^Makefile.Release, *.make)

All but one addition are full filename patterns. The
Makefile.Debug and Makefile.Release ones are used by Qt, the *.make
one is used by CMake.
---
 src/lang.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lang.c b/src/lang.c
index b0f4aec..ec39d61 100644
--- a/src/lang.c
+++ b/src/lang.c
@@ -50,7 +50,7 @@ lang_spec_t langs[] = {
     { "lisp", { "lisp", "lsp" } },
     { "lua", { "lua" } },
     { "m4", { "m4" } },
-    { "make", { "Makefiles", "mk", "mak" } },
+    { "make", { "^Makefile", "^Makefile.Debug", "^Makefile.Release", "Makefiles", "mk", "mak", "make" } },
     { "mako", { "mako" } },
     { "markdown", { "markdown", "mdown", "mdwn", "mkdn", "mkd", "md" } },
     { "mason", { "mas", "mhtml", "mpl", "mtxt" } },
-- 
2.10.1

